code/modules/mob/living/blood.dm

```diff
@@ -245,10 +245,10 @@
 
 //returns the color of the mob's blood
 /mob/living/proc/get_blood_color()
-	return "#A10808"
+	return BLOOD_COLOR_HUMAN
 
 /mob/living/carbon/xenomorph/get_blood_color()
-	return "#dffc00"
+	return BLOOD_COLOR_XENO
```

Comment:
Did we not want to disambiguate royal blood from regular blood color here? Shooting a queen makes decals colored as BLOOD_COLOR_XENO